src/components/MovieDetails/MovieDetail.js

```diff
@@ -1,26 +1,67 @@
-import './MovieDetail.css'
+import "./MovieDetail.css";
+import { useState, useEffect } from "react";
+import PropTypes from "prop-types";
 
-const MovieDetail = ({ movieDetail, goHome }) => {
-  const backgroundImageUrl = movieDetail.backdrop_path
+const MovieDetail = ({ goHome, selectedMovieId }) => {
+  const [selectedMovieDetails, setSelectedMovieDetails] = useState({});
+  const [detailsLoading, setDetailsLoading] = useState(true);
+  const [detailsError, setDetailsError] = useState(false);
+
+  const getSingleMovieData = (selectedMovieId) => {
+    fetch(
+      `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${selectedMovieId}`
+    )
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error("Server returned " + response.status);
+        }
+        return response.json();
+      })
+      .then((data) => {
+        const movieStuff = data.movie;
+        setSelectedMovieDetails(movieStuff);
+        console.log("yeehaw", data);
+      });
+    setDetailsLoading(false).catch((error) => {
+      setDetailsError(true);
+    });
+  };
+
+  useEffect(() => {
+    getSingleMovieData(selectedMovieId);
+  }, []);
+
+  const backgroundImageUrl = selectedMovieDetails.backdrop_path;
 
   const containerStyle = {
     backgroundImage: `url(${backgroundImageUrl})`,
```

Comment:
I see that you're using the style property because you need to be able to pass the backgroundImageUrl to the backgroundImage css property.  Thats a clever approach.   I'm trying to decide if I would prefer you put the remaining styles in the associated css file, or leave them altogether here to be applied via the style property.    I haven't decided yet and ultimately it is your call - just something to consider.